benchexec/tablegenerator/react-table/src/utils/filters.js

```diff
@@ -228,7 +228,11 @@ const applyMatcher = (matcher) => (data) => {
     const { value: idValue, values: idValues } = matcher.id;
     if (idValue) {
       diffd = diffd.filter(({ id }) =>
-        id.some((idName) => idName === idValue || idName.includes(idValue)),
+        id.some(
+          (idName) =>
+            idName === idValue ||
+            idName.toLowerCase().includes(idValue.toLowerCase()),
```

Comment:
possible optimization: precompute idValue.tolowerCase instead of computing it for every item